src/picker/abstractionhelper.py

```diff
@@ -1,6 +1,9 @@
 import os
+import platform
 import shutil
 
+def is_macos() -> bool:
```

Comment:
Can we please order this file alphabetically, so `is_installed`, `is_macos`, `is_wayland`?

Reply:
done